docs/usage.md

```diff
@@ -3,64 +3,63 @@
 
 # Usage
 
-Following are the methods to use FortiSOAR Health Assessment
-- Configure FortiSOAR Health Assessment Connector
-- You can use below steps to generate record
-    - **Usage 1**: Health Assessment
-        - Navigate to Health Assessment.
-        - Click on **Assess System Health**.
-        - A Health Assessment Record will be generated. [Click here](#health-assessment) for more details.
-    - **Usage 2**: Configuration Recommendation
-        - Navigate to Health Assessment.
-        - Click on **Recommend System Configuration**.
-        - A Configuration Recommendation Record will be generated. [Click here](#configuration-recommendation) for more details.
+FortiSOAR Health Assessment can be used in following ways
```

Comment:
Please elloborate with problem statement first... then usage and its benefits... and then steps.
Please add screenshots as well for important steps